### DISCUSSION TITLE
Artisan connection

### DISCUSSION THREAD

**Discussion by Khvn-01** [artisan-roaster-scope/artisan#1007] [Q&A]
Artisan requires Plc version: 6 , but may be there is some way to connect Artisan to Giesen w6-15 roaster , Plc version: 5.52 ?

**Comment by MAKOMO** [comment]
Where did you read that Artisan requires PLC version 6?

**Reply by Khvn-01** [reply]
> Good morning , Our Roaster Is Giesen w6-15(2016 with a touch screen) supported by: Siemens S7 and Plc version 5.52 . We used manual settings for Configuration IPv4 / IP adress: 192.168.2.42 / Subnet Mask: 255.255.255.0 / and used Artisan settings for machine -> Giesen -> WxA . We have been encountered an error called " Read Active Registers() S7 Communication Error: b' TCP : Connection timed out'@line 481"
If you Could help us connect Artisan , We would be very grateful to you.
                                                                                                                                            Sincerely , "Vanilla Hand Made Coffee" Company

**Reply by MAKOMO** [reply]
Is this a 6kg or a 15kg machine? You should be able to connect this machine to Artisan. The error message indicates that Artisan is not able to connect at all via TCP to the machine. Thus the network setup is not correct. Your PC is on IP address 192.168.2.42, right? I assume you used the default roasters IP address 192.168.2.180 on the Artisan setting, right? Did you also tried with 192.168.2.199? Did you check your roasters IP address it is set?

**Reply by Khvn-01** [reply]
It's a 15kg Machine. PC adress is 192.168.2.42 . We used 2 IP adresses according to instruction on the website / Default IP Addresses Giesen Roasters before 2/2018 with touch panel: W15 | 192.168.2.180 | 192.168.2.42 . We are going to try it right now (192.168.2.199)




</body>
</html>

**Reply by Khvn-01** [reply]
i.e. we can't connect because of the wrong IP address of the roaster? IP 192.168.2.199 is not working as well , encounterring the same error.
Which machine settings we should use: WxA / WxA+ / WxA Coarse / WxA IR / WxA IR Env / WxA + / WxA + IR / WxA + IR Env? We will check Roaster's IP Asap.

**Reply by bigl00z3** [reply]
Can't you see the roaster IP in one of the sub page of the control panel?

Check [this link](https://help.cropster.com/en/knowledge/how-to-find-the-ip-address-giesen)

**Reply by MAKOMO** [reply]
Use the WxA setting. However, your issue is not the exact Artisan setting but the lack of basic communication between Artisan and your roasting machine.

**Reply by Khvn-01** [reply]
We need to restart the roaster to get to sub page , unfortunatly we are not able to do it right now , because of due to the amount of work and deadlines , are we able to contact you later?

**Reply by Khvn-01** [reply]
> Use the WxA setting. However, your issue is not the exact Artisan setting but the lack of basic communication between Artisan and your roasting machine.

Are we able to solve this somehow?

**Reply by Khvn-01** [reply]
> Can't you see the roaster IP in one of the sub page of the control panel?
> 
> Check [this link](https://help.cropster.com/en/knowledge/how-to-find-the-ip-address-giesen)

Our Roaster's IP is 192.168.2.180

**Reply by MAKOMO** [reply]
..and what is your computers IP on the network interface the computer is connected to right now?

**Reply by Khvn-01** [reply]
192.168.8.198

**Reply by Khvn-01** [reply]
I have got connected TY very much

**Reply by MAKOMO** [reply]
So 192.168.8.198 is on the wrong network. It should be something like 192.168.2.x with an x different to 180.

**Reply by bigl00z3** [reply]
> Where did you read that Artisan requires PLC version 6?

It may be [here](https://artisan-scope.org/machines/giesen/)

![image](https://user-images.githubusercontent.com/8886922/202415457-eb67c823-1a92-4cd9-b01c-a6fc33b2e8bf.png)

**Reply by MAKOMO** [reply]
Oh, the 6 here stands for June not PLC v6
